### DISCUSSION TITLE
Content consumption report enable on dashboard

### DISCUSSION THREAD

**Discussion by shridhar95** [Sunbird-Obsrv/Community#127] [Issues]
Hi Team,
We have generated the content consumption report for all tenants out of which some tenant are in Live and Retired status.
Now we have requirement to make **AASAM** tenant in **Live** state which is  in **Retired** state.

We kindly request you to please help us on.

For your reference please find the attached the screenshot.
<img width="930" alt="image" src="https://github.com/Sunbird-Obsrv/Community/assets/125978181/9ecb500b-d048-4997-b087-d3cddcb27ece">

**Comment by manjudr** [comment]
@shridhar95  - This is related to learn building blocks. Kindly please check with the learn team

**Comment by manojkumarsah13** [comment]
Dear Manjunath , Thanks for your reply !! We were able to do this.  You may close this ticket.

Regards,
Manoj
